plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -110,10 +110,10 @@ public static DistributedQueryRunner createS3DeltaLakeQueryRunner(
                 coordinatorProperties,
                 extraProperties,
                 ImmutableMap.<String, String>builder()
-                        .put("hive.s3.aws-access-key", MINIO_ACCESS_KEY)
-                        .put("hive.s3.aws-secret-key", MINIO_SECRET_KEY)
-                        .put("hive.s3.endpoint", minioAddress)
-                        .put("hive.s3.path-style-access", "true")
+                        .put("aws.s3.aws-access-key", MINIO_ACCESS_KEY)
+                        .put("aws.s3.aws-secret-key", MINIO_SECRET_KEY)
```

Comment:
I would be tempted to call this just `s3.xxxx`.
did you consider that?

Reply:
I did, but I liked having the convention across all three being `cloud.service.property_name`.

Reply:
Makes sense to me.

@electrum would you mind taking a look here?